File: js/task1/S1_VAR_04.js
function typeLabel(value) {
  if (value === null) {
      return "null";
  }
  return typeof value;
}
console.log(typeLabel(null));         
console.log(typeLabel(undefined));    
console.log(typeLabel(42));          
console.log(typeLabel("42"));         
console.log(typeLabel(true));         
console.log(typeLabel({}));            
console.log(typeLabel([]));            
console.log(typeLabel(() => {}));      
console.log(typeLabel(Symbol()));

File: python/task1/S1_VAR_04.py
def typeLabel(value):
  if value is None:
      return "null"
  return type(value).__name__

print(typeLabel(None))
print(typeLabel(None))
print(typeLabel(42))
print(typeLabel("42"))
print(typeLabel(True))
print(typeLabel({}))
print(typeLabel([]))
print(typeLabel(lambda: None))
print(typeLabel(object()))
